java/anaplan-connect-library/api-core/src/main/java/com/anaplan/client/Utils.java

```diff
@@ -402,17 +369,6 @@ public static Key loadKeystore(FileInputStream fileInputStream, String keyStoreN
     return secretKeyAlias;
   }
 
-  /**
-   * Saves the provided encoded key into the keystore
-   * @param encodedKey
-   * @param keyStoreName
-   * @param password
-   * @return {@link KeyStore}
-   * @throws KeyStoreException
-   * @throws IOException
-   * @throws NoSuchAlgorithmException
-   * @throws CertificateException
-   */
   public static KeyStore saveEntryInKeyStore(byte[] encodedKey, String keyStoreName, char[] password)
```

Comment:
Please don't remove comments and java docs.

Reply:
fix